proc/process.c

```diff
@@ -395,26 +395,186 @@ static void process_tlsAssign(hal_tls_t *process_tls, hal_tls_t *tls, ptr_t tbss
 }
 
 
+static int process_validatePtr(void *map_start, size_t map_length, void *ptr_start, size_t ptr_size)
```

Comment:
Remember about camelCase ;)
```suggestion
static int process_validatePtr(void *mapStart, size_t mapLength, void *ptrStart, size_t ptrSize)
```